js/pages/estimation/components/cca-executions.js

```diff
@@ -75,7 +75,7 @@ define([
 					className: this.classes('col-exec-results'),
 					render: (s, p, d) => {
 						return (d.status === this.estimationStatusGenerationOptions.COMPLETED || d.status === this.estimationStatusGenerationOptions.FAILED) && this.isResultsViewPermitted(d.id) && d.numResultFiles > 0 ?
-							`<a href='#' data-bind="ifnot: $component.isDownloadInProgress(id), css: $component.classes('reports-link'), click: $component.downloadResults.bind($component, id)"><i class="comparative-cohort-analysis-executions__action-ico fa fa-download"></i>Download ${d.numResultFiles} files</a><span data-bind="if: $component.isDownloadInProgress(id)"><i class="prediction-generation__action-ico fa fa-spinner fa-spin"></i> Downloading ${d.numResultFiles} files...</span>`
+							`<a href='#' data-bind="ifnot: $component.isDownloadInProgress(id), css: $component.classes('reports-link'), click: $component.downloadResults.bind($component, id)"><i class="comparative-cohort-analysis-executions__action-ico fa fa-download"></i>Download</a><span data-bind="if: $component.isDownloadInProgress(id)"><i class="prediction-generation__action-ico fa fa-spinner fa-spin"></i> Downloading ${d.numResultFiles} files...</span>`
```

Comment:
You left displaying files number during downloading after the download button clicked.

Reply:
super, that is a good catch